kore/src/Kore/Logger/Output.hs

```diff
@@ -104,7 +104,7 @@ withLogger KoreLogOptions { logType, logLevel, logScopes } cont =
         LogNone     ->
             cont mempty
         LogStdOut   ->
```

Comment:
Should `LogStdOut` change to `LogStdErr` correspondingly?

Reply:
Definitely. I just read it but not see it, for some reason.